yogstation/code/modules/admin/admin_verbs.dm

```diff
@@ -57,4 +57,17 @@
 
 	var/mob/chosen_player = pick(mobs)
 	to_chat(src, "[chosen_player] has been chosen")
-	holder.show_player_panel(chosen_player)
\ No newline at end of file
+	holder.show_player_panel(chosen_player)
+
+/client/proc/get_law_history()
+	set name = "Get Law History"
+	set category = "Admin"
+	set desc = "View list of law changes for silicons."
+	var/data = ""
+	for(var/mob/living/silicon/S in GLOB.silicon_mobs)
+		data += "<h1>[S.name]:</h1>\n"
+		data += "<ol>\n"
+		for(var/lawset in S.law_history)
+			data += " <li>[lawset]</li><br>\n"
+		data += "</ol>\n"
+	src << browse(data, "window=law_history")
```

Comment:
You should probably use ``/datum/browser``.

Reply:
That would cause the popup to use tgui, I want to keep things simple and not laggy

Reply:
Lame